services/validation/README.md

```diff
@@ -1,18 +1,44 @@
 # Sourcify Script 🧑‍💻📝🔍 
 
-#### Use this script to verify if your source files are in the correct format for [Sourcify](https://github.com/ethereum/sourcify)
+The Sourcify Validation script helps you re-compile and source-verify your Solidity smart contracts.
+
+If Solidity source files are spread across multiple files, re-compiling them is often complicated.
+To help with re-compilation, the Solidity compiler generates a file called "metadata.json" that
+contains all settings, filenames and file hashes required to re-compile the contract into the exact
+same binary.
+
+Several tools, including truffle and buidler store this file with each compilation.
 
 ##### How to?
 
+Just point Sourcify Validation at your source folder that includes the build directory,
+and it will tell you how to re-compile each of your contracts:
+
 * `npm install -g sourcify-validation`
-* `sourcify-validation path/to/file1 path/to/file2 path/to/dir path/to/zip`
-    * the script will scan through all the provided files, directories and zips searching for metadata and the belonging source files
-* If everything goes well, proceed here: [Sourcify](https://verification.komputing.org/)
+* `sourcify-validation my/repository`
+
+You can specify a directories, a sequence of files and even zip files.
+The script will scan through all the provided files, search for metadata and the belonging source files.
+
+If everything goes well, you can ask it to provide a "standard-json input file"
+that you just need to send to the compiler, to etherscan, or whatever service you use
+for verification.
+
+If you use sourcify, this step is done for you. All you need to do is upload the files
+you provided to Sourcify-Validation.
+
 
 ## Example - multiple sources
 ### Command
 `sourcify-validation Escrow.sol Main.sol Owned.sol provableAPI_0.6.sol Savings.sol metadata.json`
```

Comment:
Maybe change this example to one that uses a build tool that already has the metadata somewhere?

Reply:
Ah, I see the example is the last - maybe we should combine them somehow?